app/views/ops/_settings_replication_tab.html.haml

```diff
@@ -1,207 +1,7 @@
-
-            %td{:class => "action-cell","ng-show" => "!showCancelDelete($index) && (!pglogicalReplicationModel.updateEnabled || (pglogicalReplicationModel.updateEnabled && $index !== pglogicalReplicationModel.s_index))"}
-              %button.btn.btn-default.btn-block.btn-sm{:type => "button", "ng-disabled" => "addInProgress()",  "ng-click" => "enableSubscriptionUpdate($index)"}
-                = _('Update')
-            %td{"ng-show" => "!pglogicalReplicationModel.updateEnabled || (pglogicalReplicationModel.updateEnabled && $index !== pglogicalReplicationModel.s_index)"}
-              .dropdown.pull-right.dropdown-kebab-pf
-                %button.btn.btn-link.dropdown-toggle{"data-toggle" => "dropdown", "aria-haspopup" => "true", "ng-disabled" => "showCancelDelete($index)"}
-                  %span.fa.fa-ellipsis-v
-                %ul.dropdown-menu.dropdown-menu-right.3
-                  %li
-                    %a{"ng-click" => "removeSubscription($index)"}
-                      = _('Delete')
-                  %li
-                    %a{"ng-click" => "validateSubscription($index)"}
-                      = _('Validate')
-
-            %td{:class => "action-cell", "ng-show" => "!showCancelDelete($index) && pglogicalReplicationModel.updateEnabled && $index === pglogicalReplicationModel.s_index"}
-              %button.btn.btn-default.btn-block.btn-sm{:type => "button", "ng-disabled" => "!subscriptionValid()", "ng-click" => "addSubscription($index)"}= _('Accept')
-
-            %td{"ng-show" => "pglogicalReplicationModel.updateEnabled && $index === pglogicalReplicationModel.s_index"}
-              .dropdown.pull-right.dropdown-kebab-pf
-                %button.btn.btn-link.dropdown-toggle{"data-toggle" => "dropdown", "aria-haspopup" => "true", "ng-disabled" => "showCancelDelete($index)"}
-                  %span.fa.fa-ellipsis-v
-                %ul.dropdown-menu.dropdown-menu-right.3
-                  %li
-                    %a{:class => "disabled", "ng-show" => "!subscriptionValid()"}
-                      = _('Validate')
-                  %li
-                    %a{"ng-show" => "subscriptionValid()", "ng-click" => "validateSubscription($index)"}
-                      = _('Validate')
-                  %li
-                    %a{"ng-click" => "discardSubscription($index)"}
-                      = _('Discard')
-
-    .button-group{:align => "right"}
-      %miq-button{:name      => t = _("Save"),
-                  :title     => t,
-                  :alt       => t,
-                  :enabled   => "saveEnabled(angularForm)",
-                  'on-click' => "saveClicked()"}
-      %miq-button{:name      => t = _("Reset"),
-                  :title     => t,
-                  :alt       => t,
-                  :enabled   => "saveEnabled(angularForm)",
-                  'on-click' => "resetClicked()"}
   :javascript
     ManageIQ.angular.app.value('pglogicalReplicationFormId', 'new');
     miq_bootstrap('#form_div');
```

Comment:
I think we can remove these lines too since the form is now in React, are these needed for anything on the form?